set.mm

```diff
@@ -769972,7 +769980,7 @@ _modus ponens_ (our ~ ax-mp ), but he later qualifies this.
 
 $(
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-  Formally define terms such as Reflexivity
+  Formally define notions such as reflexivity
```

Comment:
(optional, unrelated suggestion)
add ` ~ ` before the link at the bottom of this commit
![image](https://github.com/user-attachments/assets/60c342ad-6403-43bb-9c4f-8fcbf0e55196)

Reply:
Done.